pulp_ansible/app/models.py

```diff
@@ -347,6 +358,10 @@ def finalize_new_version(self, new_version):
 
                 new_version.remove_content(deprecations)
 
+    @hook(BEFORE_UPDATE, when="remote", has_changed=True)
+    def _reset_repository_last_synced_metadata_time(self):
+        self.last_synced_metadata_time = None
+
```

Comment:
I don't think this triggers when the remote is deleted. A hook on the remote being deleted is also needed I think.

Reply:
if the remote is deleted then the repo loses it's reference to it and it becomes un-syncable anyway..

Reply:
good observation